laravel-svelte-port/svelte-ui/src/routes/app/accounts/[accountId]/settings/security/+page.svelte

```
@@ -0,0 +1,98 @@
+  let savedMessage = $state<string | null>(null);
+
+  $effect(() => {
+    const preferences = authStore.uiSettings?.securityPreferences;
+    if (preferences) {
+      samlEnforced = !!preferences.samlEnforced;
+      mfaRecommended = !!preferences.mfaRecommended;
+      sessionTimeout = !!preferences.sessionTimeout;
+    }
+  });
+
+  async function savePreferences() {
```

ESLint

[error] 'savePreferences' is defined but never used.

(no-unused-vars)